CppcheckTargets.cmake

```diff
@@ -125,6 +117,11 @@ function(add_cppcheck _name)
   if(NOT TARGET cppcheck_${PROJECT_NAME})
     add_custom_target(cppcheck_${PROJECT_NAME})
   endif()
+  if(NOT TARGET ${PROJECT_NAME}_tests)
```

Comment:
target "${PROJECT_NAME}_tests" can now be added both in here and in CommonCTest.cmake, but only the latter does "set_target_properties()". Can it be added only in one place?